site/en/articles/local-fonts/index.md

```diff
@@ -0,0 +1,368 @@
+to _"only include [in the list available fonts] web fonts and fonts that come with the operating
+system, but not locally user-installed fonts"_. (And here I am, with an article on granting access
+to local fonts.)
+
+## The Local Font Access API
+
+The beginning of this article may have put you in a negative mood. Can we really not have nice
+things? Fret not. We think we can, and maybe
+[everything is not hopeless](http://hyperboleandahalf.blogspot.com/2013/05/depression-part-two.html#Blog1:~:text=like-,hopeless).
+But first, let me answer a question that you might be asking yourself.
+
+### Why do we need the Local Font Access API when there are web fonts?
+
+Professional-quality design and graphics tools have historically been difficult to deliver on the
+web. One stumbling block has been an inability to access and use the full variety of professionally
+constructed and hinted fonts that designers have locally installed. Web fonts enable some publishing
+use-cases, but fail to enable programmatic access to the vector glyph shapes and font tables used by
+rasterizers to render the glyph outlines. There is likewise no way to access a web font's binary
+data.
+
+- Design tools need access to font bytes to do their own OpenType layout implementation and allow
+  design tools to hook in at lower levels, for actions such as performing vector filters or
+  transforms on the glyph shapes.
+- Developers may have legacy font stacks for their applications that they are bringing to the web.
+  To use these stacks, they usually require direct access to font data, something web fonts do not
+  provide.
+- Some fonts may not be licensed for delivery over the web. For example, Linotype has a license for
+  some fonts that only includes [desktop use](https://www.linotype.com/25/font-licensing.html).
+
+The Local Font Access API is an attempt at solving these challenges. It consists of two parts:
+
+- A **font enumeration API**, which allows users to grant access to the full set of available system
+  fonts.
+- From each enumeration result, the ability to request low-level (byte-oriented) **SFNT container
+  access** that includes the full font data.
+
+### Current status {: #status }
+
+<div>
+
+| Step                                     | Status                                      |
+| ---------------------------------------- | ------------------------------------------- |
+| 1. Create explainer                      | [Complete][explainer]                       |
+| 2. Create initial draft of specification | [In progress][spec]                         |
+| 3. Gather feedback & iterate on design   | [Complete](#feedback).                      |
+| 4. Origin trial                          | Complete                                    |
+| 5. **Launch**                            | **Complete** (Chromium&nbsp;103 on desktop) |
+
+</div>
```

Comment:
```suggestion
```

Reply:
we can remove this given it's now launched and we have details up front of support.